apps/ledger-live-desktop/src/renderer/families/ethereum/StakeFlowModal/index.jsx

```diff
@@ -0,0 +1,36 @@
+// @flow
+
+import React, { PureComponent } from "react";
+import { Trans } from "react-i18next";
+import Modal, { ModalBody } from "~/renderer/components/Modal";
+import Body from "./Body";
+import TrackPage from "~/renderer/analytics/TrackPage";
+import { Flex } from "@ledgerhq/react-ui";
+import type { Account } from "@ledgerhq/types-live";
+
+type Props = { name: string, account: Account, checkbox?: boolean, source?: string };
+export default class DelegationModal extends PureComponent<Props> {
```

Comment:
Could this be a functional component instead of class-based?

Reply:
Agree. Done